public/pages/AnomalyCharts/containers/AnomalyHeatmapChart.tsx

```diff
@@ -228,6 +229,23 @@ export const AnomalyHeatmapChart = React.memo(
       return false;
     };
 
+    // Custom hook to refresh all of the heatmap data when running a historical task
+    useEffect(() => {
+      if (props.isHistorical) {
+        console.log('updating partial HC results');
```

Comment:
Could you provide context of this commit?

Reply:
Bug was found where historical HC results wasn't showing in the charts if the browser was left on the page without refreshing. This fixes that by updating the chart data whenever more progress is reported from the detector via the detector's `taskProgress` field.